### ISSUE TITLE
verbosity=3 ; xgboost import error

### ISSUE THREAD

**Issue by ghost** [EpistasisLab/tpot#964]
When running a default classifier keeping verbosity=3, it fails with no module named 'xgboost' error. However the same works for verbosity=2. 
Please note, we are not using xgboost classifier
## Context of the issue

Error: could not import xgboost.XGBClassifier.
No module named 'xgboost'

## Expected result

[describe what you would expect to have resulted from this process]

## Current result

Error: could not import xgboost.XGBClassifier.
No module named 'xgboost'

**Comment by weixuanfu** [comment]
Yes, the `ImportError` should be raised if the module cannot be imported but it is specified in TPOT's config when verbosity > 2.(see [those lines](https://github.com/EpistasisLab/tpot/blob/v0.11.0/tpot/operator_utils.py#L80-L83)). If you prefer using verboisty == 3 without xgboost, please check this [link](https://epistasislab.github.io/tpot/using/#customizing-tpots-operators-and-parameters)) for excluding `xgboost` from [TPOT default configuration](https://github.com/EpistasisLab/tpot/blob/master/tpot/config/classifier.py)
